daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md

```diff
@@ -54,35 +60,44 @@ spec:
 
 ### Get configuration items using gRPC API
 
-Using your [favorite language](https://grpc.io/docs/languages/), create a Dapr gRPC client from the [Dapr proto](https://github.com/dapr/dapr/blob/master/dapr/proto/runtime/v1/dapr.proto). The following examples show Java, C#, Python and Javascript clients.
+Using your [favorite language](https://grpc.io/docs/languages/), create a Dapr gRPC client from the [Dapr proto](https://github.com/dapr/dapr/blob/master/dapr/proto/runtime/v1/dapr.proto).
 
-{{< tabs Java Dotnet Python Javascript >}}
+{{< tabs Dotnet Java Python Javascript >}}
 
 {{% codetab %}}
-```java
+```csharp
 
-Dapr.ServiceBlockingStub stub = Dapr.newBlockingStub(channel);
-stub.GetConfigurationAlpha1(new GetConfigurationRequest{ StoreName = "redisconfigstore", Keys = new String[]{"myconfig"} });
+var call = client.GetConfigurationAlpha1(new GetConfigurationRequest { StoreName = "redisconfigstore", Keys = new String[]{"myconfig"} });
 ```
 {{% /codetab %}}
 
 {{% codetab %}}
-```csharp
+```java
 
-var call = client.GetConfigurationAlpha1(new GetConfigurationRequest { StoreName = "redisconfigstore", Keys = new String[]{"myconfig"} });
+Dapr.ServiceBlockingStub stub = Dapr.newBlockingStub(channel);
+stub.GetConfigurationAlpha1(new GetConfigurationRequest{ StoreName = "redisconfigstore", Keys = new String[]{"myconfig"} });
 ```
 {{% /codetab %}}
 
 {{% codetab %}}
 ```python
-response = stub.GetConfigurationAlpha1(request={ StoreName: 'redisconfigstore', Keys = ['myconfig'] })
+#dependencies
+from dapr.clients import DaprClient
+#code
+with DaprClient() as d:
+        storeName = 'configurationstore'
+        key = 'orderId'
+        d.wait(20)
```

Comment:
Can you explain why this is wait required?

Reply:
@amulyavarote, instead of wait can we use the Dapr sidecar healthcheck endpoint: https://docs.dapr.io/reference/api/health_api/#http-request

It's probably worth checking if this check is implemented in the SDK first, otherwise fallback to an http request.